MANIFEST.in

```
@@ -0,0 +1 @@
+include run_gpt/resources/*.jinja2
```

For future, it is better to include all of the files in the `resources` folder. 

```suggestion
include run_gpt/resources/*
```